.env.production

```diff
@@ -1,5 +1,17 @@
+# Node
 REACT_APP_CHAIN_ID="1"
 REACT_APP_NETWORK_URL="https://mainnet.infura.io/v3/1221fd11e90849509afafd330ec7acc6"
+
+# Wallets
 REACT_APP_PORTIS_ID="c591a488-8b86-41e1-8d2e-f4b169efd3aa"
 REACT_APP_FORTMATIC_KEY="pk_live_9846B20CC0CD7C35"
-REACT_APP_GOOGLE_ANALYTICS_ID="UA-190948266-1"
+
+# Domain
+REACT_APP_DOMAIN_PREFIX_DEV=cow-trade.dev
+REACT_APP_DOMAIN_PREFIX_STAGING=cow-trade.staging
+REACT_APP_DOMAIN_PREFIX_PROD=cow.trade
+
+# Analytics
+REACT_APP_GOOGLE_ANALYTICS_ID_DEV="UA-190948266-3"
+REACT_APP_GOOGLE_ANALYTICS_ID_STAGING="UA-190948266-4"
+REACT_APP_GOOGLE_ANALYTICS_ID_PROD="UA-190948266-1"
```

Comment:
yes, no problem being public afaik. they are per domain. So I don't think they affect us if someone is bored and want to mess up our stats